doc/changes/changes_1.0.0.md

```diff
@@ -0,0 +1,17 @@
+# Exasol Row Level Security (Lua) 1.0.0, released 2021-09-30
+
+Code name: Table filter
+
+Version 1.0.0 of Row Level Security (Lua) adds a table filter controlled by the `TABLE_FILTER` property. This allows users to selectively include tables from the source schema into the Virtual Schema.
+It helps reducing the projected tables on a need-to-know basis and speeds up meta-data scanning, since only the listed tables are scanned instead of all in the schema.
```

Comment:
Do we use hard line breaks?
```suggestion
Version 1.0.0 of Row Level Security (Lua) adds a table filter controlled by the `TABLE_FILTER` property. This allows users to selectively include tables from the source schema into the Virtual Schema. It helps reducing the projected tables on a need-to-know basis and speeds up meta-data scanning, since only the listed tables are scanned instead of all in the schema.
```